Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Hidden_User_Created.yml

```diff
@@ -0,0 +1,3098 @@
+        When more than 1000 results are retrieved, the command will return a compressed gzipped JSON format file,
+        unless the argument 'parse_result_file_to_context' is set to true and then the results will be extracted to the context.
+      script: '|||xdr-xql-generic-query'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "9"
+      '#none#':
+      - "69"
+    scriptarguments:
+      query:
+        simple: |-
+          dataset = xdr_data
+          | filter event_type = ENUM.PROCESS
+             and lowercase(agent_hostname) = lowercase("${alert.hostname}")
+             and lowercase(agent_hostname) = lowercase("${Core.OriginalAlert.raw_abioc.event.action_evtlog_data_fields.SubjectUserName}")
+          | filter action_process_image_command_line != null
+          and action_process_image_command_line != ""
+          | filter action_process_image_command_line !~= "(?i)--set-" // excludes unrelated commands
+          | fields
+              _time,
+              agent_hostname,
+              actor_effective_username,
+              action_process_image_name,
+              action_process_image_command_line
+          | comp count() as suspicious_command_count, values(action_process_image_name) as image_name, values(action_process_image_command_line) as command_line
+          by actor_effective_username
+      query_name:
+        simple: Executed Command Lines
+      time_frame:
+        simple: between ${QueryStartTime} and ${QueryEndTime}
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": 239,
+          "y": 1172
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "54":
+    id: "54"
+    taskid: e749931d-3465-4f08-8682-69c841448c6b
+    type: regular
+    task:
+      id: e749931d-3465-4f08-8682-69c841448c6b
+      version: -1
+      name: Save suspicious command lines
+      description: "Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.\n\nThis automation runs using the default Limited User role, unless you explicitly change the permissions.\nFor more information, see the section about permissions here:\n- For Cortex XSOAR 6 see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations \n- For Cortex XSOAR 8 Cloud see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script\n- For Cortex XSOAR 8.7 On-prem see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script"
```

Comment:
Should remove this as its only relevant for XSOAR:
This automation runs using the default Limited User role, unless you explicitly change the permissions.\nFor more information, see the section about permissions here:\n- For Cortex XSOAR 6 see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations \n- For Cortex XSOAR 8 Cloud see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script\n- For Cortex XSOAR 8.7 On-prem see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script"